### PR TITLE
KPI N updates

### PR DESCRIPTION
**Pull request by tomkralidis**
This PR provides updates on KPI N: DOI citation.

As always, open to comments and suggestions on how to improve this KPI (note score values, overall, are pending discussion in #71).

### CODE REVIEW

**Comment by tomkralidis** [comment]
@amilan17 @martaguti @gaubert as per today's meeting, this PR was updated to remove derefencing.

As discussed, please review and approve or request further changes.  I will update/merge thereafter.

File: KPI/N.adoc [hunk 1]
```diff
@@ -1 +1,69 @@
...
+=== WCMP element(s)
+
+* `/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation//gmd:identifier//gmd:code/gmx:Anchor/@xlink:href`
+* `/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation//gmd:identifier//gmd:code/gmx:Anchor/@xlink:title`
+* `/gmd:MD_Metadata/gmd:identificationInfo//gmd:resourceConstraints//gmd:otherConstraints/gco:CharacterString`
+
+=== What is being measured
+
+Whether DOI information is available and can be successfully identified,
+deferenced and provides citation instructions.
```
Comment (amilan17) [inline KPI/N.adoc:16]: remove 'dereferenced'

File: KPI/N.adoc [hunk 2]
```diff
@@ -1 +1,69 @@
...
+|===
+|Rule |Score
+
+|if DOI anchor exists
+|1
+
+|title is equal to 'DOI'
+|1
+
+|if DOI citation exists as a constraint, link is provided and is the same as DOI anchor
```
Comment (amilan17) [inline KPI/N.adoc:60]: not sure what 'link is provided' is looking for. a URL in the citation? If so, I don't think this is a usual practice. Recommendation: 

"if DOI citation exists as a constraint and includes the same DOI in the anchor"

**Review by amilan17 — COMMENTED** [review]
see comments inline

**Comment by tomkralidis** [comment]
Thanks @amilan17 KPI N updated per your suggestions for review/comment.

**Review by amilan17 — APPROVED** [review]

**Pull request merged**